pcsx2/R5900OpcodeImpl.cpp

```diff
@@ -975,6 +973,17 @@ void SYSCALL()
 			}
 			break;
 
+		case Syscall::Deci2Call:
```

Comment:
Some IDE's show the enum value as tooltip when hovering over the enum, I prefer having it sorted.